tests/modules/keyspace_events.c

```diff
@@ -121,6 +121,46 @@ static int KeySpace_NotificationModuleKeyMiss(RedisModuleCtx *ctx, int type, con
     return REDISMODULE_OK;
 }
 
+static int KeySpace_NotificationModuleString(RedisModuleCtx *ctx, int type, const char *event, RedisModuleString *key) {
+    REDISMODULE_NOT_USED(type);
+    REDISMODULE_NOT_USED(event);
+    RedisModuleKey *redis_key = RedisModule_OpenKey(ctx, key, REDISMODULE_READ);
+
+    size_t len = 0;
+    char *data = RedisModule_StringDMA(redis_key, &len, REDISMODULE_READ);
+    REDISMODULE_NOT_USED(data);
```

Comment:
maybe do some strncmp or alike, and add a comment that it's just to smoke out issues.?